src/pages/ResetPassword.jsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-unresolved */
+import { css } from '@emotion/react';
+import { PacmanLoader } from 'react-spinners';
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { ToastContainer } from 'react-toastify';
+import Input from '../components/Auth/PasswordInput';
+import Button from '../components/Button/PasswordButton';
+import LoginIllustation from '../assets/login-illustration.svg';
+import { processReset } from '../redux/reducers/Auth/ResetPasswordSlice';
+import LoadingSpinner from '../components/LoadingSpinner';
+
+const override = css`
```

Comment:
- there is no need of using `@emotion/react` for styling, we have tailwind for that